src/ProtocolFees.sol

```diff
@@ -53,6 +54,7 @@ abstract contract ProtocolFees is IProtocolFees, Owned {
     {
         if (msg.sender != address(protocolFeeController)) InvalidCaller.selector.revertWith();
         if (_isUnlocked()) ContractUnlocked.selector.revertWith();
+        if (!currency.isAddressZero() && CurrencyReserves.getSyncedCurrency() == currency) ProtocolFeeCurrencySynced.selector.revertWith();
```

Comment:
This is the fix for `ToB-UNI4-3`

Reply:
I believe this `==` will use `equals(Currency,Currency)` because of 
```
using {greaterThan as >, lessThan as <, greaterThanOrEqualTo as >=, equals as ==} for Currency global;`
```
but I may be wrong.